README.md

```diff
@@ -1,9 +1,6 @@
 # Envelope Zero frontend
 
-Envelope Zero is fundamentally rooted in two ideas:
-
-- Using the [envelope method](https://en.wikipedia.org/wiki/Envelope_system) to budget expenses into envelopes.
-- Zero Based Budeting, meaning that you assign all your money to an envelope. Saving for a vacation? Create an envelope and archive it after your vacation. Rent? Create an envelope that gets a fixed amount of money added every month.
+Check out the documentation at [envelope-zero.org](https://envelope-zero.org)!
```

Comment:
I would keep the little introductory paragraph and just add the link to the documentation after that.
```suggestion
Envelope Zero is fundamentally rooted in two ideas:

- Using the [envelope method](https://en.wikipedia.org/wiki/Envelope_system) to budget expenses into envelopes.
- Zero Based Budeting, meaning that you assign all your money to an envelope. Saving for a vacation? Create an envelope and archive it after your vacation. Rent? Create an envelope that gets a fixed amount of money added every month.

Check out the documentation at [envelope-zero.org](https://envelope-zero.org)!
```